### PR TITLE
Update adpp with PKARMCD variable.

### LINKED ISSUE #156: ADPP : create 6 cohorts
# TODO

1. Create a fake PKARMCD arm variable, divide the current 2 arms, into 10 groups: drug A ... J 

for PKPG01 only, first step is to do

ADPP <- ADPP %>% mutate(ARMCD = PKARMCD), so we can use 10 cohorts rather than 2.

### PR DESCRIPTION
**Pull request by ayogasekaram**
# Pull Request

Adding PKARMCD variable (results in 10 different cohorts for TLG plots).

Closes #156

### CODE REVIEW

File: R/radpp.R [hunk 1]
```diff
@@ -138,6 +138,14 @@ radpp <- function(ADSL, # nolint,
     ADPP <- mutate_na(ds = ADPP, na_vars = na_vars, na_percentage = na_percentage) # nolint
   }
 
+
+  # derive PKARMCD column for creating more cohorts
+  ADPP <- ADPP %>%
```
Comment (shajoezhu) [inline R/radpp.R:143]:
```suggestion
  ADPP <- ADPP %>% # nolint
```

**Comment by ayogasekaram** [comment]
I will wait to pull the "regimen" change to adpp after it's been merged and push to this branch again.
Thanks!

**Review by shajoezhu — APPROVED** [review]
lgtm! Thanks @ayogasekaram

**Comment by shajoezhu** [comment]
Hi @ayogasekaram , can you please merge from main, and recompile the data please. Thanks!

**Comment by shajoezhu** [comment]
lgtm! Thanks @ayogasekaram , could you recompile the doc please

```
roxygen2::roxygenize('.', roclets = c('rd', 'collate', 'namespace'))
```

**Pull request merged**